src/components/Auth/Login/Login.react.js

```diff
@@ -112,26 +112,27 @@ class Login extends Component {
             });
             let accessToken = response.access_token;
             let uuid = response.uuid;
-            let state = this.state;
             let time = response.valid_seconds;
 
-            state.isFilled = true;
-            state.accessToken = accessToken;
-            state.success = true;
-            state.msg = response.message;
-            state.time = time;
-            this.setState(state);
+            this.setState({
```

Comment:
Thanks, forgot to say this year earlier.

Please merge this setState with the one below it.

Reply:
Done @anshumanv